docs/about/contributing/modules.md

```diff
@@ -0,0 +1,395 @@
+
+## Contribute to existing modules
+
+### Types of contributions
+
+**Bug fixes**:
+
+- Fix installation or configuration issues
+- Resolve compatibility problems
+- Correct documentation errors
+
+**Feature additions**:
+
+- Add new configuration options
+- Support additional platforms or versions
+- Add new features
+
+**Maintenance**:
+
+- Update dependencies
+- Improve error handling
+- Optimize performance
+
+### Making changes
+
+1. **Identify the issue**: Reproduce the problem or identify the improvement needed
+2. **Make focused changes**: Keep modifications minimal and targeted
+3. **Maintain compatibility**: Ensure existing users aren't broken
+4. **Add tests**: Test new features and edge cases
+5. **Update documentation**: Reflect changes in the README
+
+### Backward compatibility
+
+When modifying existing modules:
+
+- Add new variables with sensible defaults
+- Don't remove existing variables without a migration path
+- Don't change variable types or meanings
+- Test that basic configurations still work
+
+## Versioning
+
+When you modify a module, update its version following semantic versioning:
+
+- **Patch** (1.0.0 → 1.0.1): Bug fixes, documentation updates
+- **Minor** (1.0.0 → 1.1.0): New features, new variables
+- **Major** (1.0.0 → 2.0.0): Breaking changes, removing variables
+
+Use the version bump script to update versions:
+
+```bash
+# For bug fixes
+./.github/scripts/version-bump.sh patch
+
+# For new features
+./.github/scripts/version-bump.sh minor
+
+# For breaking changes
+./.github/scripts/version-bump.sh major
+```
+
+## Submit your contribution
+
+1. **Create a feature branch**:
+
+   ```bash
+   git checkout -b feat/modify-git-clone-module
+   ```
+
+2. **Test thoroughly**:
+
+   ```bash
+   bun test -t 'git-clone'
+   bun fmt
+   ```
+
+3. **Commit with clear messages**:
+
+   ```bash
+   git add .
+   git commit -m "feat(git-clone):add git-clone module"
+   ```
+
+4. **Open a pull request**:
+   - Use a descriptive title
+   - Explain what the module does and why it's useful
+   - Reference any related issues
+
+## Common issues and solutions
+
+### Testing problems
+
+**Issue**: Tests fail with network errors
+**Solution**: Ensure Docker is running with `--network=host` support
+
+**Issue**: Module not found in tests
+**Solution**: Check your module path and ensure it matches the test file location
+
+### Module development
+
+**Issue**: Variables not showing in registry
+**Solution**: Ensure your README includes properly formatted variables table
+
+**Issue**: Icon not displaying
+**Solution**: Verify icon path is correct and file exists in `.icons/` directory
+
+### Documentation
+
+**Issue**: Module not appearing on registry
+**Solution**: Check frontmatter format and ensure all required fields are present
```

Comment:
This is no more valid. We now check the namespace directory to link modules to authors. 

Yes the icon could go wrong because of the front matter.